File: Pertemuan 1/Java/main.java
public class main {
    public static void main(String[] args) {
        LaptopAsus laptop = new LaptopAsus(" ROG Strix G16", "ASUS-2024-001", 16);

        System.out.println("=== Info Awal ===");
        laptop.getInfo();

        System.out.println("/n=== Operasi Sah : Nyalakan lalu pakai Baterai ===");
        laptop.nyalakan();
        laptop.pakaiBaterai(30);
        laptop.getInfo();

        System.out.println("/n=== OPerasi tidak Sah 1: Cas melebihi 100 ===");
        laptop.cashBaterai(200);
        laptop.getInfo(); // tetap maksimal 100

        System.out.println("/n=== Operasi tidak Sah 2: Nyalakan saat Baterai Habis ===");
        
        laptop.pakaiBaterai(100); // baterai jadi 0, otomatis mati
        laptop.nyalakan();
        laptop.getInfo();
    }
}

File: Pertemuan 1/php/Main.php
<?php

require_once 'LaptopAsus.php';

$laptop = new LaptopAsus("ROG Strix G16", "ASUS 2024-001", 16);

echo "=== Info Awal ===/n";
$laptop->getinfo();

echo "/n=== Operasi tidak Sah: Nyalakan lalu pakai Baterai ===/n";
$laptop->nyalakan();

$laptop->pakaiBaterai(30);
$laptop->getinfo();

echo "/n=== Operasi tidak Sah 1: Cas melebihi 100 ===/n";
$laptop->cashBaterai(200);
$laptop->getinfo(); // tetap maksimal 100

echo "/n=== Operasi tidak Sah 2: Nyalakan saat Baterai Habis ===/n";
$laptop->pakaiBaterai(100); // baterai jadi0, otomatis mati
$laptop->nyalakan();
$laptop->getinfo();